frontend/src/app/features/auth/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { addUser, login } from '@store/actions/login-page.actions';
+import { catchError, filter, map, of } from 'rxjs';
+import { getEmailValidators, getPasswordValidators } from '@utils/validators/user-validators.helper';
+
```

Comment:
Are there any logic in splitting imports or it happened accidentaly?
E,g, in our project we decied to split libraries and app item imports: at the top we have libraries, than empty line and our app imports. It is not neccessary, just wanted to understand why)

Reply:
Default linter rules have it that imports with multiple items come before ones with single item, and the extension I use for sorting imports also divides them with a blank line